runtests.py

```diff
@@ -106,6 +106,8 @@ def main(argv):
                         help="Debug build")
     parser.add_argument("--parallel", "-j", type=int, default=0,
                         help="Number of parallel jobs during build")
+    parser.add_argument("--warn-error", action="store_true",
```

Comment:
I'm not sure about making this true by default, that will be very annoying since the build isn't clean right now in many cases. And you can't expect everyone to go fix these.

Reply:
`store_true` means the default value is `False`, and when they add the `--warn-error` flag the value changes to `True`

Reply:
ah yes, never mind. been confused by that one before